portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DeleteSegments.testcase

```diff
@@ -527,8 +527,12 @@ definition {
 	}
 
 	@description = "Feature ID: Legacy | Automation ID: LRAC-8594 | Test Summary: Deleting a data source causes individuals to be removed from segment"
+	@ignore = "true"
 	@priority = "4"
 	test DeleteDataSourceIndividualsRemovedSegment {
+
+		// CSV data source tests are no longer possible to run due to this ticket LRAC-11651
+
```

Comment:
Hey @manuferreira, I think this test doesn't need to use CSV. I think it would be you have a segment with an individual and when you delete the data source, the individuals will be removed from the segment and the property. This test will still need to continue in quarantine as the membership list takes a long time to update.

I wanted @joshitagaki to confirm that this test could be done without CSV.

If he confirms, then I think you can undo this change and create a ticket for us to refactor this test or you can do it yourself in your ticket

Reply:
Correct. We don't need a CSV for this test.